test/integration/pipeline/multisig_tx_pipeline_test.cpp

```diff
@@ -157,15 +158,28 @@ TEST_F(MstPipelineTest, OnePeerSendsTest) {
       };
 
   auto &mst_itf = prepareMstItf();
-  mst_itf.sendTx(complete(tx, kUserKeypair), check_mst_pending_tx_status)
+  auto mst_tx = complete(tx, kUserKeypair);
+  mst_itf.sendTx(mst_tx, check_mst_pending_tx_status)
```

Comment:
It could be done in the separate PR, I guess.

Reply:
https://github.com/hyperledger/iroha/blob/1b1966b11214f9984f3b74a0a4c9e9f5ab352a31/test/integration/acceptance/acceptance_fixture.hpp#L22-L45 here is the example. I don't mind if it will be done in another PR, just please don't forget to put TODO there